### PR TITLE
Include Directory Needed in Newer Wiresharks

### PR DESCRIPTION
**Pull request by iguessthislldo**
To fix issue reported in https://github.com/objectcomputing/OpenDDS/discussions/3272

### CODE REVIEW

File: config/wireshark.mpb [hunk 1]
```diff
@@ -73,7 +73,7 @@ feature(wireshark) {
 feature(wireshark_cmake) {
   avoids += wireshark
 
-  includes    += $(WIRESHARK_BUILD)
+  includes    += $(WIRESHARK_BUILD) $(WIRESHARK_SRC)/include
```
Comment (mitza-oci) [inline config/wireshark.mpb:76]: Does the comment above need to be changed for this?

Comment (jwillemsen) [reply config/wireshark.mpb:76]: Is `$(WIRESHARK_SRC)/include` only for wireshark_cmake or should it be a general wireshark setting?

Comment (iguessthislldo) [reply config/wireshark.mpb:76]: > Does the comment above need to be changed for this?

`WIRESHARK_SRC` is already required by the generic base project, so I don't think so.

> Is `$(WIRESHARK_SRC)/include` only for wireshark_cmake or should it be a general wireshark setting?

No. `wireshark_cmake` is only needed when using a CMake build directory and when you install Wireshark and would need to use `wireshark_cmake=0`, the headers from `$(WIRESHARK_SRC)/include` are in the same place as before.

Comment (mitza-oci) [reply config/wireshark.mpb:76]: Right, I missed the part about it listing the "additional paths", so the one from above is still relevant for the CMake-built wireshark.

**Review by mitza-oci — APPROVED** [review]

**Pull request merged**